presto-main/src/main/java/com/facebook/presto/tdigest/TDigest.java

```diff
@@ -108,21 +111,23 @@
 
     private double totalWeight;
 
-    private final double[] weight;
+    private double[] weight;
 
-    private final double[] mean;
+    private double[] mean;
 
     private double unmergedWeight;
 
     // this is the index of the next temporary centroid
     // this is a more Java-like convention than activeCentroids uses
     private int tempUsed;
-    private final double[] tempWeight;
-    private final double[] tempMean;
+    private final int size;
```

Comment:
This name is appropriate for a local variable but seems too terse for a member variable.  How about `maxCentroidCount` and `maxBufferSize`?